### ISSUE TITLE
Internal error while parsing invalid code

### ISSUE THREAD

**Issue by leonardoalt** [ethereum/act#73]
```
constructor of LValue
interface constructor()

creates

	uint x
```
```
constructor of LValue
interface constructor()

creates

	uint
```
both lead to
```
Internal error
no valid tokens
```

**Comment by leonardoalt** [comment]
Maybe fixed by https://github.com/ethereum/act/pull/68

**Comment by xwvvvvwx** [comment]
hmmm unlikely fixed by #68, this is an error from the parser. Agree that it would be nice to have some position information here also....

**Comment by MrChico** [comment]
This stems from https://github.com/ethereum/act/blob/master/src/Parse.y#L310. And I think the problem here is that the parser expected more tokens, but there weren't any more tokens. But since the error is the lack of a token I don't think we can get any position information from `parseError` at least. We could create a special case of `AlexPosn` or something to indicate the last token maybe
